### PR TITLE
Tcp udp transport

### PR DESCRIPTION
**Pull request by DoctorVoid**

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -18,6 +18,7 @@
     "rules": {
         "@typescript-eslint/explicit-function-return-type": "off",
         "@typescript-eslint/no-explicit-any": "off",
-        "@typescript-eslint/no-var-requires": "off"
+        "@typescript-eslint/no-var-requires": "off",
+        "@typescript-eslint/explicit-module-boundary-types": "off"
```
Comment (chenshoo) [inline .eslintrc:22]: why

File: src/transports/dynamic-logstash-transport.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import { LogstashProtocol } from '../configurations/logstash-protocol';
+import { LogstashOption, LogstashTransport } from 'winston-logstash-ts';
+
+const sizeof = require('object-sizeof');
```
Comment (DoctorVoid) [inline src/transports/dynamic-logstash-transport.ts:4]: no types library?

Comment (ronkatz96) [reply src/transports/dynamic-logstash-transport.ts:4]: no :(

**Pull request merged**